Sources/MediaEventProcessor.swift

```diff
@@ -145,11 +137,17 @@ class MediaEventProcessor: MediaEventProcessing {
             return
         }
 
-        session.abort {
-            self.mediaSessions.removeValue(forKey: sessionId)
-            Log.trace(label: Self.LOG_TAG, "[\(Self.CLASS_NAME)<\(#function)>] - Successfully aborted media session (\(sessionId)).")
-        }
+        session.abort()
+        self.mediaSessions.removeValue(forKey: sessionId)
```

Comment:
Can we have the logic to remove all the inactive sessions. So we clear out any other session which maybe inactive nut not deleted as we do on the Android side.
`self.mediaSessions.forEach(self.removeInactiveSession)`

Reply:
It actually is removing all sessions. This private `abort` function is called from `abortAllSessions` which loops through the `mediaSessions` dictionary to abort and remove all sessions.